packages/editor-ui/src/components/FilterConditions/Condition.vue

```diff
@@ -105,13 +105,13 @@ const rightParameter = computed<INodeProperties>(() => {
 
 const debouncedEmitUpdate = debounce(() => emit('update', condition.value), { debounceTime: 500 });
 
-const onLeftValueChange = (update: IUpdateInformation<NodeParameterValue>): void => {
```

Comment:
Wouldn't leaving the generic here avoid casting `as NodeParameterValue`?

Reply:
I typed the emit that this function listens to with `IUpdateInformation`, so leaving the generic gives an error